tools/cli/commands.go

```diff
@@ -0,0 +1,871 @@
+	FlagDomain                    = "domain"
+	FlagDomainWithAlias           = FlagDomain + ", do"
+	FlagCustomizeBuilder          = "customize_builder"
+	FlagCustomizeBuilderWithAlias = FlagCustomizeBuilder + ", cb"
+	FlagWorkflowID                = "workflow_id"
+	FlagWorkflowIDWithAlias       = FlagWorkflowID + ", wid, w"
+	FlagRunID                     = "run_id"
+	FlagRunIDWithAlias            = FlagRunID + ", rid, r"
+	FlagTaskList                  = "tasklist"
+	FlagTaskListWithAlias         = FlagTaskList + ", tl"
+	FlagWorkflowType              = "workflow_type"
+	FlagWorkflowTypeWithAlias     = FlagWorkflowType + ", wt"
+	FlagExecutionTimeout          = "execution_timeout"
+	FlagExecutionTimeoutWithAlias = FlagExecutionTimeout + ", et"
+	FlagDecisionTimeout           = "decision_timeout"
+	FlagDecisionTimeoutWithAlias  = FlagDecisionTimeout + ", dt"
+	FlagInput                     = "input"
+	FlagInputWithAlias            = FlagInput + ", i"
+	FlagInputFile                 = "input_file"
+	FlagInputFileWithAlias        = FlagInputFile + ", if"
+	FlagReason                    = "reason"
+	FlagReasonWithAlias           = FlagReason + ", re"
+	FlagOpen                      = "open"
+	FlagOpenWithAlias             = FlagOpen + ", op"
+	FlagPageSize                  = "pagesize"
+	FlagPageSizeWithAlias         = FlagPageSize + ", ps"
+	FlagEarliestTime              = "earliest_time"
+	FlagEarliestTimeWithAlias     = FlagEarliestTime + ", et"
+	FlagLatestTime                = "latest_time"
+	FlagLatestTimeWithAlias       = FlagLatestTime + ", lt"
+	FlagPrintRawTime              = "print_raw_time"
+	FlagPrintRawTimeWithAlias     = FlagPrintRawTime + ", prt"
+	FlagDescription               = "description"
+	FlagDescriptionWithAlias      = FlagDescription + ", desc"
+	FlagOwnerEmail                = "owner_email"
+	FlagOwnerEmailWithAlias       = FlagOwnerEmail + ", oe"
+	FlagRetentionDays             = "retention_days"
+	FlagRetentionDaysWithAlias    = FlagRetentionDays + ", rd"
+	FlagEmitMetric                = "emit_metric"
+	FlagEmitMetricWithAlias       = FlagEmitMetric + ", em"
+	FlagName                      = "name"
+	FlagNameWithAlias             = FlagName + ", n"
+	FlagEnv                       = "env"
+	FlagOutputFilename            = "output_filename"
+	FlagOutputFilenameWithAlias   = FlagOutputFilename + ", of"
+	FlagQueryType                 = "query_type"
+	FlagQueryTypeWithAlias        = FlagQueryType + ", qt"
+)
+
+const (
+	localHostPort = "127.0.0.1:7933"
+
+	maxOutputStringLength = 200 // max length for output string
+
+	defaultTimeFormat          = time.RFC3339 // used for converting UnixNano to string like 2018-02-15T16:16:36-08:00
+	defaultDomainRetentionDays = 3
+)
+
+// For color output to terminal
+var colorRed func(i ...interface{}) string
+var colorMagenta func(i ...interface{}) string
+var colorGreen func(i ...interface{}) string
+
+func init() {
+	colorRed = color.New(color.FgRed).SprintFunc()
```

Comment:
could you assign the value when you declare the variables so we don't need this init()?

Reply:
changed